sale_commission/wizard/wizard_settle.py

```diff
@@ -32,35 +39,43 @@ class SaleCommissionMakeSettle(models.TransientModel):
     agents = fields.Many2many(comodel_name='res.partner',
                               domain="[('agent', '=', True)]")
 
-    def _get_period_start(self, agent, date_to):
+    @classmethod
```

Comment:
Instead of @classmethod, put this as @api.model methods (it's the recommended way for methods that doesn't matter the value of self - https://www.odoo.com/documentation/8.0/reference/orm.html#openerp.api.model).